indico/modules/events/timetable/controllers/manage.py

```diff
@@ -112,3 +113,21 @@ def _process_PATCH(self):
             with track_time_changes():
                 update_timetable_entry(self.timetable_entry, updates)
         return jsonify()
+
+    def _process_DELETE(self):
+        """Delete a timetable entry"""
+        delete_timetable_entry(self.timetable_entry)
+
+
+class RHTimetableBalloon(RHTimetableREST):
+    def _process(self):
+        if not self.timetable_entry:
+            raise BadRequest('Required key missing')
+        html = None
+        if self.timetable_entry.contribution:
```

Comment:
most likely not ;) just wanted to point it out. in the end it doesn't really matter here since it's it's nothing where you would query things in a loop so an extra query or two doesn't make any significant difference